risk-assessment/__tests__/index.test.ts

```diff
@@ -0,0 +1,174 @@
+import './setup';
+import * as core from '@actions/core';
```

Comment:
Usually, we keep tests next to src files. Same comment in another PR

Reply:
Fine by me. I prefer having tests in __test__ directories too. At least you are aware that the standard is different in Hawk.